esx_service/utils/auth_api.py

```diff
@@ -432,13 +454,24 @@ def _tenant_vm_add(name, vm_list):
         error_info = error_code.generate_error_info(ErrorCode.VM_ALREADY_IN_TENANT, vm_list, name)
         return error_info
 
+    error_info, tenant_list = get_tenant_list_from_db()
```

Comment:
@shaominchen  makes sense. 

So when the vmlist is empty for vm add operation, do we return without error? Or should we return an error enforcing the user to specify list of VMs when triggering vm add operation to a tenant?